Braintree.xcodeproj/project.pbxproj

```diff
@@ -1613,6 +1623,7 @@
 				804698292B27C4D70090878E /* BraintreeShopperInsights */,
 				A91E1EAC24FEEF21007540E5 /* BraintreeThreeDSecure */,
 				A7C889F01B5F0B30007A0E9C /* BraintreeVenmo */,
+				621964B82DDCF82300684434 /* Helpers */,
```

Comment:
For directory and pattern lets use the same as is being used for `VenmoQueryPaymentContextGraphQLBody`, example here: https://github.com/braintree/braintree_ios/pull/1569/files

We can make a `Models` directory and will want to make sure we update the path for cocoapods, if we havent already. Then naming wise can call the file <FEATURE><GQL REQUEST NAME>GraphQLBody. Happy to pair on this too if you'd like.